### DISCUSSION TITLE
how to create a react app?

### DISCUSSION THREAD

**Discussion by liyanage1234** [generate-all/generate-all-in-one#10] [Q&A] [answered]
how to create a react app?

**Comment by Heshan-Lahiru** [comment, marked as answer]
Install Node.js and npm
node -v
npm -v

Install Create React App
npm install -g create-react-app

Create a New React App
npx create-react-app my-app

Navigate to the Project Folder
cd my-app

Start the development server:
npm start

Project Structure
my-app/
├── node_modules/
├── public/
│   ├── index.html
├── src/
│   ├── App.js
│   ├── index.js
├── package.json
├── .gitignore

Building Your App
npm run build

**Reply by liyanage1234** [reply]
thanks
